CadRevealComposer/SceneCreator.cs

```diff
@@ -169,7 +169,7 @@ private static SectorInfo SerializeSector(InternalSector p, string outputDirecto
 
         string? sectorFilename = !p.Geometries.Any()
             ? null
-            : (!p.IsHighlightSector ? $"sector_{p.SectorId}.glb" : $"highlight_sector_{p.SectorId}.glb"); // The highlight_sector name is used in the frontend to identify the highlight sectors, don't rename without syncing with the frontend.
+            : (!p.IsPrioritizedSector ? $"sector_{p.SectorId}.glb" : $"highlight_sector_{p.SectorId}.glb"); // The highlight_sector name is used in the frontend to identify the highlight sectors, don't rename without syncing with the frontend.
```

Comment:
Should the filename be updated to `priority_sector_...` as well?

Reply:
yes, but we currently have a guard in the client that checks for the "highlight"-prefix, which needs to be changed first. We do have a task for whitelisting instead of blacklisting, which should handle it 🙂